tracker-reporting/utils.js

```diff
@@ -0,0 +1,10 @@
+// Delays a function in ms based on a URL param delay=x
+// eslint-disable-next-line no-unused-vars
+function callAfterDelay (callback) {
+    if (location.search.includes('delay')) {
+        const delay = location.search.match(/[&?]delay=([0-9]+)/);
+        setTimeout(callback, delay[1]);
```

Comment:
Let's make this code POP by using SearchParams.

```suggestion
    const params = (new URL(location.href)).searchParams;
    const delay = parseInt(params.get('delay'), 10) || 0;

    if (delay) {
        setTimeout(callback, delay);
```